platforms/ios/example/.swiftformat

```diff
@@ -1,5 +1,6 @@
 --swiftversion 5.6
 
+--disable redundantStaticSelf
```

Comment:
Perhaps worth adding a link to the related issue here (unless we plan on keeping this rule disabled forever)?

Reply:
[ae07ae3](https://github.com/matrix-org/matrix-rich-text-editor/pull/773/commits/ae07ae314af7e10ac9835c1c6b029afe33205811)